css/defold.scss

```diff
@@ -20,7 +19,120 @@
 
 	--defold-orange: #fd6623;
 }
+$color-darker: #191d21;
+$color-lighter: #afb9c4;
 
+/*******************************************************************************
+* dark mode
+******************************************************************************/
+
+.dark-mode {
+    .dark\:dark {
+        background-color: var(--dark) !important;
+        color: var(--lighter) !important;
+
+		a {
+			color: var(--lighter);
+		}
+		a.anchor-link {
+			opacity: 0.3;
+
+			&:hover {
+				opacity: 0.7;
+			}
+		}
+		a:hover {
+			color: var(--lightest);
+		}
+		h1 {
+			color: var(--lightest);
+		}
+		h2 {
+			color: var(--lightest);
+		}
+		h3 {
+			color: var(--lightest);
+		}
+		h4 {
+			color: var(--lightest);
+		}
+		h5 {
+			color: var(--lightest);
+		}
+		.card {
+			background-color: var(--grey);
+
+			a {
+				text-decoration: none;
+				color: var(--lightest);
+
+				&:hover {
+					text-decoration: underline;
+					color: var(--lightest);
+				}
+			}
+		}
+		code, .example pre, span.type {
+			background-color: var(--darker);
+			border: 1px solid lighten($color-darker, 10%);
+		}
+		span.type {
+			color: darken($color-lighter, 10%);
+		}
+		.sidenote {
+			background-color: #574910;
+			color: var(--almostlightest);
+		}
+		.important {
+			background-color: #591515;
+			color: var(--almostlightest);
+		}
+		input[type="text"] {
+			background-color: var(--darker);
+		}
+		img.logo-hor-classic-160 {
+			content: url("/images/logo/defold/logo_with_text/logo-hor-classic-white-160.png");
+		}
+    }
+
+    .dark\:darkest {
```

Comment:
`dark:dark` and `dark:darkest` overlap. There must be a way to not duplicate the styles but I'm not an expert in SCSS...